deploy/common/k8sbased.sh

```diff
@@ -194,8 +194,8 @@ function launch_central {
     	add_args "--with-config-file=${ROXDEPLOY_CONFIG_FILE_MAP}"
     fi
 
-    if [[ "$POD_SECURITY_POLICIES" == "true" ]]; then
-      add_args "--enable-pod-security-policies"
+    if [[ "$POD_SECURITY_POLICIES" == "false" ]]; then
+      add_args "--enable-pod-security-policies=false"
     fi
```

Comment:
Please do this:

```suggestion
    if [[ -n "$POD_SECURITY_POLICIES" ]]; then
      add_args "--enable-pod-security-policies=${POD_SECURITY_POLICIES}"
    fi
```

so that the flag gets a value whenever `$POD_SECURITY_POLICIES` is set and we don't implicitly rely on flag's default value to be `true` because it may change.